packages/cardano-services-client/src/HandleProvider/KoraLabsHandleProvider.ts

```diff
@@ -44,10 +44,12 @@ export const toHandleResolution = ({
   apiResponse: IHandle;
   policyId: Cardano.PolicyId;
 }): HandleResolution => ({
+  backgroundImage: Asset.Uri(`ipfs://${apiResponse.background}`),
```

Comment:
The handle image is returned with the ipfs protocol prefix. It would make sense that `profilePic` and `backgroundImage` would too.  Do you know if this could result in` ipfs://ipfs://...`?

Reply:
@lucas-barros there's a different between `image`, `backgroundImage` and `profilePic`, as the latter two are from the custom metadata, whereas the former is the standard NFT metadata.

Reply:
@rhyslbw makes sense. I see the scheme on public API specifies it as only the ipfs hash.

Reply:
@rhyslbw Actually the API response was already in the codebase [here](https://github.com/input-output-hk/cardano-js-sdk/blob/1efd9862686136fd5c106ddc56e126e8b0da8868/packages/cardano-services-client/test/util.ts#L46-L67). I confirmed that it matches the data in the api [docs](https://koralabs.github.io/handles-public-api/#/handles/get_handles__handle_).